server/src/jobs/fiabilisation/uai-siret/build.ts

```diff
@@ -210,6 +215,38 @@ const getAllCouplesUaiSiretTdb = async () => {
     .toArray();
 };
 
+const getAllCouplesUaiSiretTdbInReferentiel = async () => {
+  return await effectifsDb()
+    .aggregate([
+      { $match: filters },
+      {
+        $lookup: {
+          from: "organismes",
+          localField: "organisme_id",
+          foreignField: "_id",
+          as: "organismes_info",
+          pipeline: [
+            {
+              $match: {
+                $expr: {
+                  $or: [
+                    { $eq: ["$est_dans_le_referentiel", STATUT_PRESENCE_REFERENTIEL.PRESENT] },
+                    { $eq: ["$est_dans_le_referentiel", STATUT_PRESENCE_REFERENTIEL.PRESENT_UAI_MULTIPLES_TDB] },
+                  ],
+                },
```

Comment:
Bonus : Potentiellement, ça pourrait être un peu plus court avec un $in (et faire la même chose).
```js
est_dans_le_referentiel: {
   $in: [STATUT_PRESENCE_REFERENTIEL.PRESENT, STATUT_PRESENCE_REFERENTIEL.PRESENT_UAI_MULTIPLES_TDB]
}
```

J'ai fait un petit test en local chez moi. Et en fait le $expr + $or est nettement plus long. 24s au lieu de 17s. C'est pas très impactant ici mais faut en avoir conscience :smile:

Reply:
Bien vu, je vais update 🙏🏼